src/step_definitions/elements.ts

```diff
@@ -3,9 +3,10 @@ import { When, Then } from 'cucumber';
 import { comparators } from '../comparators';
 import config from '../core/config.helper';
 import { matchers, regexBuilder } from '../matchers';
-import { waitForCondition } from '../web/cucumber/wait-for-condition.helper';
+import { waitForCondition } from '../web/methods/wait-for-condition.helper';
 import variableStore from '../core/variable-store.helper';
 import { createValueToTextTransformer } from '../transformers/transformer/values-to-text.transformer';
+import { interactions } from '../web/methods/interactions';
```

Comment:
I think that we should import `kakMethods` from `../web/methods/index.ts`... later on there won't be just `interactions.ts`